Avoid setting Python libraries as install targets to avoid DESTDIR
installation issues.
--- cmake/NektarCommon.cmake.orig	2020-07-10 10:19:58.000000000 +0100
+++ cmake/NektarCommon.cmake	2020-07-10 10:20:47.000000000 +0100
@@ -268,6 +268,8 @@
     # Python requires a .so extension, even on OS X.
     SET_TARGET_PROPERTIES(_${name} PROPERTIES PREFIX "")
     SET_TARGET_PROPERTIES(_${name} PROPERTIES SUFFIX ".so")
+    SET_TARGET_PROPERTIES(_${name} PROPERTIES
+        LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/NekPy/${name})
 
     ADD_DEPENDENCIES(_${name} boost-numpy)
 
@@ -287,7 +289,6 @@
     SET(TMPOUT "${TMPOUT}from ._${name} import *")
 
     FILE(WRITE ${CMAKE_BINARY_DIR}/NekPy/${name}/__init__.py ${TMPOUT})
-    INSTALL(TARGETS _${name} DESTINATION ${CMAKE_BINARY_DIR}/NekPy/${name})
 ENDMACRO()
 
 MACRO(ADD_NEKPY_EXECUTABLE name source)
